src/components/ebay-line-chart/component.ts

```diff
@@ -40,6 +40,7 @@ interface LineChartInput
     "cdn-highcharts-accessibility"?: string;
     version?: string;
     series: SeriesLineOptions | SeriesLineOptions[];
+    outside?: boolean;
```

Comment:
Since this "outside" property pertains to the tooltip configuration, I'm wondering if this should be renamed to something like `tooltipOutside` or `renderTooltipOutside`... I'm not quite sure what would make the most sense

Reply:
I was struggling back and forth on that naming while working on this. I had `renderTooltipOutside` initially, and I think it makes sense to go back to that flag naming.